File: POS.Admin/ClientApp/src/app/modules/district/models/country-model.ts
export class CountryModel {
    countryId: number;
    countryName: string;
    countryNameAr: string;
    currencyId: number;
    code: string;
    flag: string;
    currencyName: string;
    currencyNameAr: string;
}

export class CityModel {
    cityId: number;
    cityName: string;
    cityNameAr: string;
    countryId: number;
    countryName: string;
    countryNameAr: string;
    insertedBy: string;
    createDate: string;
    lastModifyDate: string;
    modifiedBy: string;
}

export class DistrictModel {
    districtId: number;
    districtName: string
    districtNameAr: string
    cityId: number;
    inActive: boolean;
    cityName: string
    cityNameAr: string;
    countryId:number;
    insertedBy: string;
    createDate: string;
    lastModifyDate: string;
    modifiedBy: string;
}

File: NSR.API/Models/CountryModel.cs
using NSR.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace NSR.API.Models
{
  public class CountryModel
    {
        public int CountryId { get; set; }
        public string CountryName { get; set; }
        public string CountryNameAr { get; set; }
        public int? CurrencyId { get; set; }
        public string Code { get; set; }
        public string Flag { get; set; }
        public string CurrencyName { get; set; }
        public string CurrencyNameAr { get; set; }
    }
}
